### PR TITLE
ENT-5351: fix: Made the median_posting_duration in JobPosting Nullable to avoid errors on jenkins.

### PR DESCRIPTION
**Pull request by saleem-latif**
__Description:__
Management command to refresh job postings data on the Jenkins server was failing, this PR has made the `JobPosting.median_posting_duration` nullable to avoid unnecessary errors.

**Merge checklist:**
- [x] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `make upgrade && make requirements` have been run to regenerate requirements
- [x] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
- [x] [Version](https://github.com/edx/taxonomy-connector/blob/master/taxonomy/__init__.py) bumped
- [x] [Changelog](https://github.com/edx/taxonomy-connector/blob/master/CHANGELOG.rst) record added

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/taxonomy-connector/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/edx/taxonomy-connector/actions), verify version has been pushed to [PyPI](https://pypi.org/project/taxonomy-connector/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [course-discovery](https://github.com/edx/course-discovery) to upgrade dependencies (including taxonomy-connector)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in course-discovery will look for the latest version in PyPi.

### CODE REVIEW

File: test_settings.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def root(*args):
 # Settings related to to EMSI client
 # API URLs are altered to avoid accidentally calling the API in tests
 # Original URL: https://auth.emsicloud.com/connect/token
-EMSI_API_ACCESS_TOKEN_URL = 'http://example.com/connect/token'
+EMSI_API_ACCESS_TOKEN_URL = 'https://auth.emsicloud.com/connect/token'
```
Comment (muhammad-ammar) [inline test_settings.py:52]: why this change?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 --------------------
 
+[1.14.6] - 2022-02-14
+---------------------
+
+* fix: Made the median_posting_duration in JobPosting Nullable tp avoid errors on jenkins.
```
Comment (HammadAhmadWaqas) [inline CHANGELOG.rst:20]: Nit: type: tp -< to

File: taxonomy/signals/handlers.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
 @receiver(UPDATE_COURSE_SKILLS)
 def handle_update_course_skills(sender, course_uuid, **kwargs):  # pylint: disable=unused-argument
     """
-    Handle signal and trigger task to update course skills.
+    Handle signal and trigger tas k to update course skills.
```
Comment (HammadAhmadWaqas) [inline taxonomy/signals/handlers.py:19]: why this change?

**Review by muhammad-ammar — APPROVED** [review]

**Review by HammadAhmadWaqas — APPROVED** [review]
Few typos: otherwise good
